kanopi/saplings-component-types
# TW29378047 Explo Eclipse Hero

## Description
Teamwork Ticket(s): [Explo Eclipse Hero](https://kanopi.teamwork.com/app/tasks/29378047)

> Adds config for the Hero component for Saplings

## Acceptance Criteria
* Video or image can be added to the Hero component
* Config does not have UUIDs
* Field names start with 'sa_'

## Steps to Validate
1. Add a Hero component
2. Save and confirm it functions as expected
